lib/params.ex

```diff
@@ -98,10 +98,7 @@ defmodule Strukt.Params do
     end
   end
 
-  defp get_struct_field_value(struct, field) do
-    case struct do
-      nil -> nil
-      struct -> Map.get(struct, field)
-    end
-  end
+  # defp get_struct_field_value(nil, _field), do: nil
+
+  defp get_struct_field_value(struct, field), do: Map.get(struct, field)
```

Comment:
This change introduces a bug: `Map.get/2` will blow up if `struct` is `nil`, which is why the original code matches `struct` defensively first.